### ISSUE TITLE
Tacotron2 Pre-training have difficulties

### ISSUE THREAD

**Issue by gyu-bbang** [TensorSpeech/TensorFlowTTS#782]
Hello, I am a student who is learning with the Tacotron2 Kss dataset.

If you proceed with Tacotron2 Kss pre-training 120k and check the results through the tensor board, the following result values are given.

The loss percentage in the "val" section tends to be higher and higher.

If you pull it out as a wav file, the sound quality is indistinguishable.

I'd like to ask for your advice on this matter.
![Screenshot from 2022-12-20 16-09-30](https://user-images.githubusercontent.com/97422685/208607996-21c89851-35c6-4148-9f35-78ff41d75bd0.png)
